File: src/windows/magma/node/JavaStringType.c
#include "JavaStringType.h"
#include "../../java/util/Optional.h"
/*public */struct JavaStringType {/*
*/};
/*@Override
    public */struct CType toCType_JavaStringType() {
	return new_Pointer(CPrimitive.Char);
}
/*@Override
    public */struct Optional_char_ptr findBaseName_JavaStringType() {
	return empty_Optional();
}
/*
*/

File: src/java/magma/node/JavaStringType.java
package magma.node;

import java.util.Optional;

public class JavaStringType implements JavaType {
    @Override
    public CType toCType() {
        return new Pointer(CPrimitive.Char);
    }

    @Override
    public Optional<String> findBaseName() {
        return Optional.empty();
    }
}
